code/game/turfs/open/space/transit.dm

```diff
@@ -6,37 +6,53 @@
 	flags_1 = NOJAUNT_1 //This line goes out to every wizard that ever managed to escape the den. I'm sorry.
 	explosion_block = INFINITY
 
+/turf/open/space/transit/Initialize(mapload)
+	. = ..()
+	update_appearance()
```

Comment:
calling update_appearance on initialize is extremely wasteful, I changed this in a PR to just do the transform change since that is all that's actually needed. you might also need to set the icon state.

Reply:
it uses update_icon now; does that work?